ui/server/src/main/resources/db/migration/postgres/V1_028__AddMissingArchiveAction.sql

```diff
@@ -0,0 +1,21 @@
+ALTER TABLE "process_actions" DROP CONSTRAINT "process_deployment_info_check";
```

Comment:
Is there any difference between this and h2 version? If not, let's put it to common?

Reply:
Oo  good catch! I didn't know about it.. ;)